scripts/ci/determine_and_run_tests.sh

```diff
@@ -3,7 +3,9 @@
 echo "Running $TEST_SUITE"
 
 if [[ $TEST_SUITE = "Kubernetes"  ]]; then
-  go test -timeout 50m -v ./test/minikube
+  gotestsum --junitfile /tmp/test-results/gotestsum-report-${CIRCLE_BUILD_NUM}.xml --format testname -- ./test/minikube
```

Comment:
What is `gotestsum`?

Reply:
gotestsum runs tests using go test -json, prints formatted test output, and a summary of the test run. Basically, we will have the test report using gotestsum.